vyper/venom/function.py

```diff
@@ -142,7 +124,7 @@ def _compute_reachability(self) -> None:
         """
         Compute reachability of basic blocks.
         """
-        for bb in self.basic_blocks:
+        for bb in self._basic_block_dict.values():
```

Comment:
Use `get_basic_blocks()` for uniformity?

Reply:
https://github.com/vyperlang/vyper/pull/4002/commits/f8238c3670691907bc68c013bdbf0a189896661e